### DISCUSSION TITLE
Why some live blogs show as empty feed?

### DISCUSSION THREAD

**Discussion by baannng** [FreshRSS/FreshRSS#3484] [General]
hi
I've been adding some old blogs to my list and the result is Empty Feed.
Like https://flowingdata.com/ or https://coolinfographics.com/blog

How could I fix this?

Thank you.

**Comment by Alkarex** [comment]
Both https://flowingdata.com/feed and https://coolinfographics.com/blog?format=rss work for me, and are not empty.
Please check your logs (FreshRSS logs, Web server logs), as there might be some errors there.
Both feeds have some recent articles.

So for those feeds, that is not the problem, but to answer the original question, in order to work with feeds, which only have old articles, make sure that the "Maximum age of articles to keep" is old enough:

![image](https://user-images.githubusercontent.com/1008324/109355827-1cd80c00-7880-11eb-878d-5d2c61a8fa98.png)

**Comment by baannng** [comment]
Hi Alkarex
I unselected all purge policies so they can keep the feeds.
I have these errors: 
2021-03-08 18:43:21 in_player_instance does not exist in configuration
2021-03-08 18:43:21 in_show_content does not exist in configuration
2021-03-08 18:43:21 in_player_height does not exist in configuration
2021-03-08 18:43:21 in_player_width does not exist in configuration
2021-03-08 18:33:54 Unknown error for feed [https://coolinfographics.com/blog]
2021-03-08 18:33:24 A feed could not be found at `https://coolinfographics.com/blog`; the status code is `200` and content-type is `text/html;charset=utf-8` [https://coolinfographics.com/blog]

I've changed the feed link to https://coolinfographics.com/blog?format=rss and still doesn't download the feeds.

BTW is it possible to reset the DB with losing the subscription list and categories?

Could you help?
Thank you.

**Reply by Alkarex** [reply]
https://coolinfographics.com/blog?format=rss works for me. Try to delete the feed if you have it already and add it again.
If that still fail, try with https://coolinfographics.com/blog?format=rss&2

To reset your database, there are a few options. For instance, you can use the export menu (in particular the OPML), make a new user with admin rights, and import again. If everything works, you can delete the old user.

**Comment by baannng** [comment]
Hi Alkarex
Thank you for your help.
I couldn't solve the issue so I had to reset totally my installation.
BTW, I will open 2 tickets about installation on server and image loading :)
Thank you once again.
AMP.
